muvsfunc.py

```diff
@@ -5692,17 +5695,18 @@ def exclusive_accumulate(iterable: Iterable[T], func: Callable[[T, T], T] = oper
     # Output spliced clip
     output = core.std.Splice(clips, mismatch=True)
 
-    # Use a reference clip for CFR output.
-    if cfr_output:
+    # For CFR output.
+    if cfr_ref:
         clipa = output
-        clipb = ref
-        fpn_out = clipa.num_frames*clipb.fps.numerator/clipb.num_frames
-        fpd_out = clipb.fps.denominator
-        return core.std.AssumeFPS(output, fpsnum=fpn_out, fpsden=fpd_out)
+        clipb = cfr_ref
+        fpn = clipa.num_frames * clipb.fps.numerator
+        fpd = clipb.fps.denominator * clipb.num_frames
+        return core.std.AssumeFPS(output, fpsnum=fpn, fpsden=fpd)
+    elif cfr_ref == False:
+        return core.std.AssumeFPS(output, fpsnum=fpsnum_out, fpsden=fpsden_out)
```

Comment:
What's the purpose of this line of code? It breaks existing scripts that use this function, I think.

Reply:
```python
    '''
        cfr_ref: (bool or VideoNode)
            You can input a reference clip (can be your source clip) for CFR output.
            Or you can set False and use custom FrameRate Numerator & FrameRate Denominator for CFR output.
            If you input a reference clip, it should also be CFR, and the time of the output clip should same as the reference clip.
            Default: None.
    '''
```
The purpose is setting your _custom FrameRate Numerator & FrameRate Denominator for CFR output._
As `None`  forced, the output is same as original codes, unless set it to `False` or input a clip.

Reply:
This functionality can be implemented outside the function, and the two extra parameters can be removed for simplicity.

Reply:
> This functionality can be implemented outside the function, and the two extra parameters can be removed for simplicity.

Ok.